src/sql/src/plan/statement/ddl.rs

```diff
@@ -1922,6 +1981,8 @@ fn kafka_sink_builder(
     let consistency_topic = consistency_config.clone().map(|config| config.0);
     let consistency_format = consistency_config.map(|config| config.1);
 
+    // XXX(chae): is this where secrets should be inlined for kafka sinks??  "End of planning"?
```

Comment:
similar flag here: inlining _could_ be delayed by passing through a `SecretsReader` ref/copy into `KafkaSinkConnectorBuilder` which the sink can use on its own startup -- but it's unclear to me if that small delay is worth the overhead of plumbing through `SecretsReader`

Reply:
and also here